utils/scripts/ocm/ocm.py

```diff
@@ -658,6 +658,26 @@ def wait_for_osd_cluster_to_get_deleted(self, timeout=3600):
                    " EXITING".format(self.cluster_name))
             sys.exit(1)
 
+    def update_notification_email_address(self, addon_name, email_address):
+        """Update notification email to Addons"""
+        replace_vars = {
+                       "EMAIL_ADDER": email_address
+                       }
```

Comment:
Shouldn't it be EMAIL_ADDRE or EMAIL_ADDRESS ?

Reply:
I just kept it as short intentionally> It is just a simple variable for render